CONTRIBUTING.md

```diff
@@ -164,6 +165,48 @@ This repository contains several scripts and commands for performing numerous ta
 
 </details>
 
+## Experimental Cloudflare Deployment
```

Comment:
I've remmemerd to address https://github.com/nodejs/nodejs.org/pull/7383#discussion_r1978174650 😄👍